### DISCUSSION TITLE
Geomap route with grouped data

### DISCUSSION THREAD

**Discussion by HenrikRolandHansen** [grafana/grafana#56940] [Dashboards, Panels & Visualizations]
The Geomap does currently not support grouping of the route data. So if several traces from different vessels are plotted the route will connect all positions as one route. Would be nice if it was possible to get different routes and coloring based on grouping. 

![image](https://user-images.githubusercontent.com/8969659/195825925-21a5e6b8-0e95-4ed2-a609-e58f4664887c.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
